Sources/Proxy/Representations/SliceRepresentationProxy/index.js

```diff
@@ -255,10 +255,13 @@ function vtkSliceRepresentationProxy(publicAPI, model) {
     const bds = ds.getBounds();
     const axisIndex = 'XYZ'.indexOf(slicingMode);
     const endValue = bds[axisIndex * 2 + 1];
+    const ijkAxis = model.mapper.getClosestIJKAxis().ijkMode;
+    const isXYZ = 'XYZ'.indexOf(model.mapper.getSlicingMode()) !== -1;
```

Comment:
isn't
```js
'XYZ'.indexOf(model.mapper.getSlicingMode())
```
the same as
```js
const axisIndex = 'XYZ'.indexOf(slicingMode);
```
above?

Reply:
Also seems like all those values could be computed generically from looking at the content of `getState` which is computed by `updateDomains`?